Revisions of citizen science (#55)
PathOS-project/indicator_handbook: indicator_templates/quarto/1_open_science/citizen_science.qmd

The European Citizen Science Association ECSA [@citizen2018] published ten principles describing the citizen science approach and the first five constitute an excellent definition of this approach:

> 1.  Citizen science projects actively involve citizens in scientific endeavors that generate new knowledge or understanding. Citizens may act as contributors, collaborators or as project leaders and have a meaningful role in the project.

Author: Probably stick to the UK spelling

```suggestion
1.  Citizen science projects actively involve citizens in scientific endeavours that generate new knowledge or understanding. Citizens may act as contributors, collaborators or as project leaders and have a meaningful role in the project.
```

-   the conservation and sharing of the resources generated by the project \[8. ownership\]
-   the recognition as authors/protagonists of the research \[9. credit\]

This document describes a series of metrics to quantify (but also qualify) each of these nine different forms of citizen participation in science, as well as a tenth indicator accounting for the capacity of a citizen sciences project to span across multiple forms of participation \[10. Span\].

## Open Science in European research programmes

> Citizen Science is possibly the most studied part of Open Science and one that has been heavily supported by the European Union in the FP7 and Horizon 2020 programs. At the time of writing (november 2024), a search for Citizen Science in the Cordis database (https://cordis.europa.eu) returns 93 programmes and 443 projects.

Author: Some minor editorial suggestions:

```suggestion
Citizen Science is possibly the most studied aspect of Open Science and one that has been heavily supported by the European Union in the FP7 and Horizon 2020 programs. At the time of writing (November 2024), a search for Citizen Science in the Cordis database (https://cordis.europa.eu) returns 93 programmes and 443 projects.
```

Reviewer: Agreed!

> Beyond direct project funding, the EU also supports citizen science through organizations like the European Citizen Science Association (ECSA), which plays an important role as the hub for the scientific community using this approach [@vohland_citizen_2021]. 

Author: Minor editorial suggestion

```suggestion
Beyond direct project funding, the EU also supports citizen science through organizations such as the European Citizen Science Association (ECSA), which plays an important role as the hub for the scientific community using this approach [@vohland_citizen_2021]. 
```

Reviewer: Agreed!

> Founded in 2013 ECSA supports a network of citizen science initiatives, promoting high standards, shared methodologies, and collaboration opportunities. Its platform is centralize training material, resources, and guidelines for citizen scientists. 

Author: Unclear phrasing, see suggested fix below, but I'm not 100% sure if that is what you want to say.

```suggestion
Founded in 2013 ECSA supports a network of citizen science initiatives, promoting high standards, shared methodologies, and collaboration opportunities. Its platform centralises training material, resources, and guidelines for citizen scientists. 
```

Reviewer: Agreed!

> In the United Kingdom a similar role has been played by the Open Air Laboratories (OPAL) since  2007. Focussing on environmental monitoring, OPAL promotes education and awareness.and has influenced national environmental policy and public awareness about air, soil, and water quality.

Author: Fix a typo

```suggestion
In the United Kingdom a similar role has been played by the Open Air Laboratories (OPAL) since 2007. Focussing on environmental monitoring, OPAL promotes education and awareness and has influenced national environmental policy and public awareness about air, soil, and water quality.
```

-   The most straightforward way of measuring this is by the simple count of the number of steps in which citizens have been given a chance to be active.
-   A less binary option is to define a scale of "citizen agency/participation" for each step and then compute the average value across the whole research protocol (see for instance the model proposed by [@gharesifard2017].

> ##

Author: I guess this can just be removed?

Reviewer: I thought you needed for the publication online but, if not, go ahead and remove
